ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -238,7 +239,16 @@ private KafkaStreams buildStreams(
       newStreamsProperties.put(
           StreamsConfig.DEFAULT_TIMESTAMP_EXTRACTOR_CLASS_CONFIG, KsqlTimestampExtractor.class);
     }
-    return new KafkaStreams(builder.build(), new StreamsConfig(newStreamsProperties));
+    Topology build = builder.build();
+    //    TopologyDescription describe = build.describe();
```

Comment:
i guess you didn't mean to leave this here

Reply:
should the name be `topology` instead of `build`?